markdown/ch8.markdown

```diff
@@ -1,6 +1,18 @@
 # Big Data in Little Laptop with Toolz
 
-Whenever I think too hard about streaming data analysis, my head hurts.
+> GRACIE: A knife? The guy's twelve feet tall!
+> JACK: Seven. Hey, don't worry, I think I can handle him.
+>
+> — Jack Burton, Big Trouble in Little China
+
+Although streaming is not really a SciPy feature, it is a way of writing code
```

Comment:
How about this mix? I want to stress the "science" angle to justify talking about it in a SciPy book. =)

> Although streaming is not a SciPy feature per se, it is a way of processing data
> that is critical to efficiently processing the large datasets that we see in
> science.

I also don't like "resource constrained machines" — every machine is resource constrained given a large enough dataset...